core/src/main/scala/org/bykn/bosatsu/LetFreeEvaluation.scala

```diff
@@ -32,4 +32,12 @@ object LetFreeEvaluation {
 
   type Cache = Option[CMap[String, (Future[Value], Type)]]
   type ToLFV = Option[LetFreeValue => Future[Value]]
+
+
+  def exprFn(wrapper: (LetFreeEvaluation.LetFreeValue, rankn.Type, LetFreeEvaluation.Cache, LetFreeEvaluation.ToLFV) => Any): FfiCall = {
```

Comment:
we move the function from Ffi here so we keep value agnostic of as much as possible.

Reply:
This is much nicer here